sensor_msgs/include/sensor_msgs/image_encodings.hpp

```diff
@@ -241,9 +240,7 @@ static inline int bitDepth(const std::string & encoding)
     return std::atoi(m[0].str().c_str());
   }
 
-  if (encoding == YUV422 ||
-    encoding == YUV422_YUY2 ||
-    encoding == UYVY ||
+  if (encoding == UYVY ||
```

Comment:
We can't remove the implementation at the same time as the deprecation. They need at least a full release cycle before being removed. And the cost of keeping them here is so low I'm not sure it makes sense to remove at all.

Reply:
Should I then just keep the "deprecation" in the PR and leave the usage of the deprecated encodings in the encoding checks as they were? This would mean that you will get a lot of deprecation warnings when this header is used. If this is ok, I can revert this partially.